posts/2021-04-19-JAXRS-multipart-payloads-21005-beta.adoc

```diff
@@ -0,0 +1,168 @@
+---
+layout: post
+title: "Building Multipart payloads for JAX-RS client and services in Open Liberty 21.0.0.5-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: Building Multipart payloads for JAX-RS client and services in Open Liberty 21.0.0.5-beta - OpenLiberty.io
+seo-description: With Open Liberty 21.0.0.5-beta you can now build multipart payloads for you JAX-RS client and services and also we now offer Jakarta EE 9 Support for `oauth-2.0` and `samlWeb-2.0`.  
+blog_description: With Open Liberty 21.0.0.5-beta you can now build multipart payloads for you JAX-RS client and services and also we now offer Jakarta EE 9 Support for `oauth-2.0` and `samlWeb-2.0`.  
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Building Multipart payloads for JAX-RS client and services in Open Liberty 21.0.0.5-beta
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+
+With Open Liberty 21.0.0.5-beta you can now build multipart payloads for you JAX-RS client and services and also we now offer Jakarta EE 9 Support for `oauth-2.0` and `samlWeb-2.0`.
```

Comment:
With Open Liberty 21.0.0.5-beta your can now build multipart payloads for **your** JAX-RS client and services.  We also now offer Jakarta EE 9 **support** for `oauth-2.0` and `samlWeb-2.0`.

Reply:
Done